homeassistant/scripts/macos/__init__.py

```diff
@@ -54,12 +54,14 @@ def run(args):
     if args[0] == "install":
         install_osx()
         return 0
+
     if args[0] == "uninstall":
         uninstall_osx()
         return 0
-    if args[0] == "restart":
-        uninstall_osx()
-        # A small delay is needed on some systems to let the unload finish.
-        time.sleep(0.5)
-        install_osx()
-        return 0
+
+    # last case is "restart"
```

Comment:
This is dangerous IMHO, as it assume it will be restart.

The proper handling here (IMHO), is raising when an unknown arg is received.

Reply:
> This is dangerous IMHO, as it assume it will be restart.
> 
> The proper handling here (IMHO), is raising when an unknown arg is received.

I did that way originally because there is a check above with the available commands.

I have adjusted it anyway to raise.